src/constraint/constrSys.ml

```diff
@@ -31,7 +31,7 @@ type 'v sys_change_info = {
 }
 
 (** A side-effecting system. *)
-module type MonSystem =
+module type MonadicSystem =
```

Comment:
Renamed so that it is obvious from the name that this does not relate to monotonicity.